flume/neore/scripts/package.sh

```diff
@@ -87,7 +87,7 @@ function copy_cygnus_to_flume(){
 	mkdir -p ${RPM_PRODUCT_SOURCE_DIR}/plugins.d/cygnus
 	mkdir ${RPM_PRODUCT_SOURCE_DIR}/plugins.d/cygnus/lib
 	mkdir ${RPM_PRODUCT_SOURCE_DIR}/plugins.d/cygnus/libext
-	cp $BASE_DIR/target/cygnus-${PRODUCT_VERSION}-jar-with-dependencies.jar ${RPM_PRODUCT_SOURCE_DIR}/plugins.d/cygnus/lib
+	cp $BASE_DIR/target/cygnus-*-jar-with-dependencies.jar ${RPM_PRODUCT_SOURCE_DIR}/plugins.d/cygnus/lib
```

Comment:
This is dangerous... if the target directoy is not perefectly clean, you will end with several .jar packages in the RPM. In fact, I think such situation happended in the past, so we use a concrete version in the jar file.